554747 - net-snmp-config should not contain perl options

Remove rpath from net-snmp-config --agent-libs output.

diff -up net-snmp-5.5/net-snmp-config.in.jsaf net-snmp-5.5/net-snmp-config.in
--- net-snmp-5.5/net-snmp-config.in.jsaf	2009-04-04 12:07:39.000000000 +0200
+++ net-snmp-5.5/net-snmp-config.in	2010-08-19 16:08:28.825276858 +0200
@@ -50,7 +50,7 @@ NSC_LDFLAGS="@LDFLAGS@"
 NSC_INCLUDEDIR=${includedir}
 NSC_LIBDIR=-L${libdir}
 NSC_LIBS="@LIBS@"
-NSC_AGENTLIBS="@AGENTLIBS@ @PERLLDOPTS_FOR_APPS@"
+NSC_AGENTLIBS="@AGENTLIBS@"
 NSC_PREFIX=$prefix
 NSC_EXEC_PREFIX=$exec_prefix
 NSC_SRCDIR=@srcdir@
